repos/fdbt-site/src/pages/api/getExportProgress.ts

```diff
@@ -27,6 +29,12 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
         const noc = getAndValidateNoc(req, res);
 
         const exportNames = await getS3Exports(noc);
+        const selectExportAttribute = getSessionAttribute(req, SELECT_EXPORTS_ATTRIBUTE);
+        const currTime = new Date().getTime() / 1000;
```

Comment:
if I run new Date().getTime(), I get 1677164819135 . If I divide that by 1000, i get 1677164825.979 , which will always be more than 5.

Reply:
Also, why is this code in the getExportProgress API? We need to be doing this inside exports.tsx

Reply:
@NitishAroraKPMG ok but the comment above is asking why this is in the getExportProgress API?